scripts/justinstall.sh

```diff
@@ -0,0 +1,123 @@
+cd $WORKSPACE
+ALTISCALE_RELEASE=${ALTISCALE_RELEASE:-0.1.0}
```

Comment:
Yes, However, most of our build script has the default as 0.1.0. I will change it as 4.0.0.